lib/db/mysql.js

```diff
@@ -916,15 +916,12 @@ module.exports = function (log, error) {
 
   // Update : emails
   // Values : uid = $1, email = $2
-  var SET_PRIMARY_EMAIL = 'CALL setPrimaryEmail_3(?, ?)'
+  var SET_PRIMARY_EMAIL = 'CALL setPrimaryEmail_4(?, ?)'
   MySql.prototype.setPrimaryEmail = function (uid, email) {
-    return this.write(
-      SET_PRIMARY_EMAIL,
-      [
-        uid,
-        email
-      ]
-    )
+    return this.read(SET_PRIMARY_EMAIL, [uid, email])
```

Comment:
Why did this change from `this.write()` to `this.read()`?

Reply:
Ah, I might have missed this when testing different approaches. I'll update to `this.write()`.